knodle/transformation/majority.py

```diff
@@ -34,6 +35,30 @@ def probabilies_to_majority_vote(
         raise ValueError("Specify a way how to resolve unclear majority votes.")
 
 
+def z_matrix_to_rule_idx(
```

Comment:
it is currently not used anywhere, right? will you somehow need it later?

Reply:
I do use it at the moment in my project - but I did not implement the adversarial code to Knodle by now.